src/main/java/lotto/utils/LottoMatchStatus.java

```diff
@@ -0,0 +1,22 @@
+package lotto.utils;
+
+public enum LottoMatchStatus {
+    THREE_MATCH_RESULT("3개 일치 (5,000원)", 5000),
+    FOUR_MATCH_RESULT("4개 일치 (50,000원)", 50000),
+    FIVE_MATCH_RESULT("5개 일치 (1,500,000원)", 1500000),
+    FIVE_MATCH_WITH_BONUS_RESULT("5개 일치, 보너스 볼 일치 (30,000,000원)", 30000000),
+    SIX_MATCH_RESULT("6개 일치 (2,000,000,000원)", 2000000000);
```

Comment:
자바에서 숫자를 사용할때 구분자로 "_ "를 사용할 수 있습니다 이를 통해 더 직관적으로 숫자를 확인할 수 있습니다. 이는 코드 가독성을 위한 표시일 뿐 실제 값에는 영향을 주지 않습니다.
```suggestion
    SIX_MATCH_RESULT("6개 일치 (2,000,000,000원)", 2_000_000_000);
```